pkg/capabilities/triggers/mercury_trigger.go

```diff
@@ -99,12 +99,7 @@ func (o *MercuryTriggerService) RegisterTrigger(ctx context.Context, req capabil
 		return nil, err
 	}
 
-	inputs, err := o.ValidateInputs(req.Inputs)
-	if err != nil {
-		return nil, err
-	}
-
-	triggerID := o.getTriggerID(inputs.TriggerID, wid)
+	triggerID := o.getTriggerID(req.TriggerID, wid)
```

Comment:
If this is already unique by workflow, could this just be
```suggestion
	triggerID := req.TriggerID
```